### PR TITLE
Pyic 7344 strategic app api tests

### PR DESCRIPTION
**Pull request by DanCorderIPV**
### What changed

Create API tests to replace strategic app end to end tests

### Why did it change

API tests are quicker to run and more reliable

### Issue tracking

- [PYIC-7344](https://govukverify.atlassian.net/browse/PYIC-7344)

[PYIC-7344]: https://govukverify.atlassian.net/browse/PYIC-7344?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api-tests/features/p2-strategic-app.feature [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@core-back-api-test
```
Comment (MikeCollingwood) [inline api-tests/features/p2-strategic-app.feature]: This tag is different to other tests, which have `@Build`. What was the reason for this?

File: local-running/core.local.params.yaml [hunk 2]
```diff
@@ -113,7 +113,7 @@ core:
         stub: '{
           "credentialUrl": "https://dcmaw-async.stubs.account.gov.uk/async/credential",
           "tokenUrl":"https://dcmaw-async.stubs.account.gov.uk/async/token",
-          "clientId": "ipv-core-local",
+          "clientId": "dummyClientId",
```
Comment (MikeCollingwood) [inline local-running/core.local.params.yaml:116]: Did this change for a reason?

**Review by MikeCollingwood — COMMENTED** [review]
Just a couple of clarifying questions

File: api-tests/features/p2-strategic-app.feature (hunk 1 above)
Comment (A-Rashid9) [reply api-tests/features/p2-strategic-app.feature]: @MikeCollingwood  I believe these were changed to so we can run the tag  @core-back-api-test

Comment (DanCorderIPV) [reply api-tests/features/p2-strategic-app.feature]: I misunderstood another PR I saw - reverted

File: local-running/core.local.params.yaml (hunk 2 above)
Comment (DanCorderIPV) [reply local-running/core.local.params.yaml:116]: Yes, this is now the correct value for the build environment.

**Review by MikeCollingwood — APPROVED** [review]

**Pull request merged**